src/shared/awsContextCommands.ts

```diff
@@ -3,20 +3,12 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import globals from './extensionGlobals'
-
 import * as vscode from 'vscode'
 import * as nls from 'vscode-nls'
 const localize = nls.loadMessageBundle()
 
-import { LoginManager } from '../credentials/loginManager'
-import { asString, CredentialsId, fromString } from '../credentials/providers/credentials'
-import { CredentialsProviderManager } from '../credentials/providers/credentialsProviderManager'
+import { asString, CredentialsId } from '../credentials/providers/credentials'
 import * as extensionConstants from './constants'
-import {
-    credentialProfileSelector,
```

Comment:
`DefaultCredentialSelectionDataProvider` is dead code now
Need to add the 'open ~/.aws/credentials' action somewhere in the switch connection picker